openedx/features/edly/api/serializers.py

```diff
@@ -25,7 +25,8 @@ def get_app_config(self, obj):
             url = self.context['site_configuration'].get('SITE_NAME', '')
             protocol = 'https' if self.context['request'].is_secure() else 'http'
             mobile_app_config['API_HOST_URL'] = '{}://{}'.format(protocol, url) if url else ''
-            mobile_app_config['ORGANIZATION_CODE'] = self.context['edly_sub_org_of_user'].edx_organization.short_name
+            mobile_app_config['ORGANIZATION_CODE'] = self.context[
```

Comment:
Nit: fix the formatting issue

Reply:
Quality test failing due to line length. Thats why I need to split